.storybook/themes/news.scss

```diff
@@ -1 +1,192 @@
-@use "@wpmedia/arc-themes-components/scss";
+@use "@wpmedia/arc-themes-components/scss" with (
+	$breakpoints: (
+		default: "min-width: 0",
+		desktop: "min-width: 48rem",
+	),
+	$global: (
+		"spacing-1": 0.25rem,
+		// 4px
+		"spacing-2": 0.5rem,
+		// 8px
+		"spacing-3": 0.75rem,
+		// 12px
+		"spacing-4": 1rem,
+		// 16px
+		"spacing-5": 1.5rem,
+		// 24px
+		"spacing-6": 2rem,
+		// 32px
+		"spacing-7": 2.5rem,
+		// 40px
+		"spacing-8": 3rem,
+		// 48px
+		"spacing-9": 3.5rem,
+		// 56px
+		"spacing-10": 4rem,
+		// 64px
+		"spacing-11": 4.5rem,
+		// 72px
+		"spacing-12": 5rem,
+		// 80px
+		"spacing-13": 5.5rem,
+		// 88px
+		"spacing-14": 6rem,
+		// 96px
+		"spacing-15": 6.5rem,
+		// 104px
+		"spacing-16": 7rem,
+		// 112px
+		"spacing-17": 7.5rem,
+	),
+	$alias: (
+		// Layout
+		"layout-max-width": 1440,
+		"content-max-width": 1216,
```

Comment:
The max-width for News theme is 1440px - this setting is showing a lot of visual regression changes via chromatic

Reply:
Thanks for looking at this and storybook. Not sure I totally follow your comment. I have the alias set as 1440. What are you suggesting that I do?

Reply:
I can replace what I have with content-scale-width 1440 explicitly. is that the goal?

Reply:
Updated @matthewroach

Reply:
@JackHowa - I created a separate pull request to show what I was meaning. https://github.com/WPMedia/arc-themes-blocks/pull/1342

The "container" for news themes is 1440px - and then there is spacing either side of the content which means the overall layout width needs to wider for the `content-scale-width` logic to work to apply the responsive space either side of the container

You can see the issue in the Chromatic snapshot from this branch - https://www.chromatic.com/test?appId=5f91e4721ccaba0022a10782&id=627ae0fd101555004a8c62bd

and on the new pull request for the same chromatic snapshot you can see the spacing either side of the layout is maintained - https://www.chromatic.com/test?appId=5f91e4721ccaba0022a10782&id=627b79425d891d004aa05e06

Reply:
ah I see at the 1200 px breakpoint. it looks like the spacing on the sides are improved in the new snapshot
![Screen Shot 2022-05-11 at 07 00 42](https://user-images.githubusercontent.com/5950956/167844918-e144b55e-9c9d-4645-8026-7baa0160e123.png)

Reply:
merged